core/domain/story_services_test.py

```diff
@@ -561,7 +561,7 @@ def test_get_pending_nodes_in_story(self):
 
         for index, pending_node in enumerate(
                 story_services.get_pending_nodes_in_story(
-                    self.owner_id, self.STORY_1_ID), 1):
+                    self.owner_id, self.STORY_1_ID), start=1):
```

Comment:
I'm going to repeat a previous comment:

> That said, this seems odd. Just use index + 1 below perhaps (and explain what's going on in a comment if you think it may be confusing).

Perhaps I should clarify that the above entails not using the "start" arg. And you might want a comment to explain why we are using "index + 1" in that case, since counting typically starts from 0.

Reply:
Done.